packages/svelte/src/compiler/phases/3-transform/server/visitors/SvelteElement.js

```diff
@@ -47,6 +47,7 @@ export function SvelteElement(node, context) {
 					'$.push_element',
 					b.id('$$payload'),
 					tag,
+					b.literal(false),
```

Comment:
Couldn't this also be a custom element?

```svelte
<svelte:element tag={tag} is="my-element" />
```
i wonder if we need to add extra checks for the SvelteElement case (before it was using the name so it was probably fine)?